simplified-app-ekirjasto/src/main/assets/txnative/sv/txstrings.json

```diff
@@ -898,7 +1165,7 @@
          "string" : "Ok"
       },
       "placeholder" : {
-         "string" : ""
+         "string" : "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eu leo laoreet, mattis augue ac, placerat nibh. Sed odio nunc, tempor sed justo eu, condimentum eleifend lectus. Sed at leo iaculis, euismod neque nec, rhoncus diam. Pellentesque molestie hendrerit massa ac cursus. Proin sed feugiat mi. Morbi sollicitudin, nulla vitae eleifend laoreet, ligula dolor pretium nunc, ac feugiat nibh sapien vitae augue. Pellentesque ut sollicitudin magna. Ut ac purus pellentesque, venenatis turpis id, auctor augue. Aliquam bibendum lectus id urna placerat auctor. Nullam faucibus molestie neque. Etiam in ligula eget tellus dictum ullamcorper volutpat vel erat. Mauris ut nulla urna."
```

Comment:
This should be removed from all languages in Transifex, the original Android string has been removed, but it still exists in Transifex.

Reply:
Cannot be removed through the Transifex UI due to license restrictions, but should be possible to remove via the API, I'll handle these separate of this PR.